src/box/lua/iproto.c

```diff
@@ -119,6 +153,17 @@ push_iproto_key_enum(struct lua_State *L)
 		{"VCLOCK_SYNC", IPROTO_VCLOCK_SYNC},
 	};
 	push_iproto_constant_subnamespace(L, "key", keys, lengthof(keys));
+	for (size_t i = 0; i < lengthof(keys); ++i) {
+		size_t len = strlen(keys[i].name);
+		const char *name = constant_tolower(keys[i].name, len);
+		struct mh_strnu32_node_t node = {
+			.str = name,
+			.len = len,
+			.hash = mh_strn_hash(name, len),
+			.val = keys[i].val,
+		};
+		mh_strnu32_put(iproto_key_translation, &node, NULL, NULL);
+	}
```

Comment:
IMO it's strange that the aliases are only available for common keys.
For example, the user might want to form a Raft packet or a ballot, in which case he'll need IPROTO_RAFT_* or IPROTO_BALLOT_* keys.

Reply:
This would significantly complicate the implementation, for we'd have to take into account the context when doing substitution, while the benefit of this isn't clear: overriding the Raft implementation isn't the primary goal of this API. We decided that we won't implement it for now. We may, however, implement it later, if there's a justified request for this.